tests/tt_metal/tt_metal/unit_tests_fast_dispatch/common/command_queue_fixture.hpp

```diff
@@ -62,3 +62,30 @@ class MultiCommandQueueFixture : public ::testing::Test {
     tt::ARCH arch_;
     size_t num_devices_;
 };
+
+class CommandQueueMultiDeviceFixture : public ::testing::Test {
```

Comment:
whats the difference between this and `MultiCommandQueueFixture` and also is it just me or is `num_devices_` undefined?

Reply:
I think MultiCommandQueueFixture is gone now. And yes, good catch. Fixed.